client/keys/show.go

```diff
@@ -142,6 +142,22 @@ func runShowCmd(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
+func fetchKey(kb keyring.Keybase, keyref string) (keyring.Info, error) {
+	info, err := kb.Get(keyref)
```

Comment:
If I am not wrong it does not return error if the key does not exist.

Reply:
Ah ok, it only happens with dbKeyBase, but is confusing then.

Reply:
It does, try with `gaiacli keys show SOMEINVALIDKEY`

Reply:
You are right!